androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -219,6 +224,20 @@ public void destroy() {
         mDestroyed = true;
     }
 
+    public static final FallbackStrategy cctFalbackStrategy =
```

Comment:
Could you move this to the top (with the definition of Fallback Strategy)?

Also, static final variables should be BLOCK_CAPS.

Reply:
done